README.md

```diff
@@ -210,6 +246,7 @@ build-time configuration, environment variables, and expected artifacts.
 
 ---
 
+<!-- deprecate devkit -->
```

Comment:
It's as much a note to myself as anything else right now; I might remove the commit entirely but I didn't want to forget that this is something I wanted to do.